Dockerfile.compile

```diff
@@ -5,4 +5,4 @@ RUN apk add --update \
 LABEL maintainer="sig-platform@spinnaker.io"
 ENV GRADLE_USER_HOME /workspace/.gradle
 ENV GRADLE_OPTS -Xmx4g
-CMD ./gradlew -PenableCrossCompilerPlugin=true --no-daemon gate-web:installDist -x test
+CMD ./gradlew --no-daemon gate-web:installDist -x test
```

Comment:
There are more places to remove in github actions...

Reply:
THINK i've caught em all now :)